lib/l10n/app_en.arb

```diff
@@ -81,6 +81,28 @@
             "capacity": {}
         }
     },
+    "oath_custom_icons": "Custom icons",
+    "oath_custom_icons_description": "Icon packs can make your accounts more easily distinguishable with familiar logos and colors.",
+    "oath_custom_icons_replace": "Replace icon pack",
+    "oath_custom_icons_loading": "Icon pack is loading...",
+    "oath_custom_icons_load": "Load icon pack",
```

Comment:
Maybe it's worth pointing out somewhere that we are expecting a .zip file? For example perhaps just `Load icon pack (.zip)` would do fine?

Reply:
Maybe, but is it really needed? The file dialog will only show .zip files as selectable, but if we're thinking about the flow you first downloaded the icon package from somewhere, it doesn't really matter what the extension is, you downloaded a file and that's the one you're going to be looking for.

Reply:
I think the filter in the file picker is probably sufficient. I would change "Icon pack is loading..." to "Loading icon pack..." though.

Reply:
I changed the _Loading_ copy to suggested string and kept the _Load_ copy without the (.zip).